src/docker/tss/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# syntax=docker/dockerfile:1-labs
+
+# Aptos CLI git metadata.
+ARG GIT_REPO=https://github.com/aptos-labs/aptos-core.git
+
+# Relative to Econia repository root.
+ARG MOVE_ROOT=src/move
+ARG ECONIA_ROOT=$MOVE_ROOT/econia
+ARG FAUCET_ROOT=$MOVE_ROOT/faucet
+ARG ACCOUNT_FILES=src/docker/tss/accounts/
+
+# Relative to image root.
+ARG NODE_BIN=/aptos-core/target/cli/aptos
+
+# Get Aptos repo and build dependencies, then compile.
+FROM rust:1-bookworm AS compile-cli
+ARG GIT_REPO
+ARG NODE_BIN
+RUN git clone \
+    $GIT_REPO \
+    --depth 1
```

Comment:
This should use a git tag because sometimes the head of `main` is unstable

Reply:
done